massa-client/src/rpc.rs

```diff
@@ -112,6 +112,15 @@ impl RpcClient {
         self.call_method("unban", "()", vec![ips]).await
     }
 
+    pub(crate) async fn node_whitelist(&self, ips: Vec<IpAddr>) -> RpcResult<()> {
+        self.call_method("node_whitelist", "()", vec![ips]).await
+    }
```

Comment:
Just a naming proposition, (sorry I'm late :pray:)  `append_whitelist_ip`

Reply:
it's because it works the same as ban and unban